content/docs/concepts/security/index.md

```diff
@@ -287,6 +287,10 @@ When a client calls the `datastore` service, it extracts the `test-team` identit
 and checks whether `test-team` is allowed to run `datastore` with the secure naming information.
 The client detects that `test-team` is **not** allowed to run the `datastore` service and the authentication fails.
 
+Secure naming is able to protect against general network hijackings for HTTPS traffic. However, it would fail to work 
```

Comment:
Please cite network hijacking examples: 
... against network hijackings (i.e. DNS spoofing, BGP/route hijacking, ARP spoofing, etc.) for HTTPS traffic. It can also protect TCP traffic from network hijackings except DNS spoofing.

Reply:
Done